cypress/spec/create_work.cy.js

```diff
@@ -0,0 +1,57 @@
+describe('Create work', () => {
+    let work_id
+
+    beforeEach(() => {
+      const results = cy.appFactories([
+        ['create', 'work_version_with_work_and_collection', 'with_required_associations', {} ]
+      ]).then((results) => {
+        cy.log(results[0])
```

Comment:
cruft

Reply:
Actually this is part of the test -- the fixture sets up a draft work with all of the required metadata filled in, so all that is left to do to complete the deposit and make it valid is upload a file.  If we didn't do this, we'd have to tediously fill in a big form.